File: sdk/alcedo-sdk/src/appSettings.ts
export function createAppSettingsResource(ky: any) {
  return {
    list: (options?: any) => ky.get("settings", options).json(),
    update: (key: string, value: any, options?: any) =>
      ky.put(`settings/${encodeURIComponent(key)}`, { json: { value }, ...options }).json(),
    batch: (settings: Record<string, any>, options?: any) =>
      ky.post("settings/batch", { json: { settings }, ...options }).json(),
  };
}

File: sdk/alcedocore-sdk-python/alcedo_sdk/resources/app_settings.py
from typing import Any, Optional

from .base import BaseResource


class AppSettingsResource(BaseResource):
    async def list(self) -> dict:
        return await self._request("GET", "/api/app-settings")

    async def update(self, key: str, value: Any) -> dict:
        return await self._request("PUT", f"/api/app-settings/{key}", json={"value": value})

    async def batch(self, settings: dict) -> dict:
        return await self._request("PUT", "/api/app-settings", json={"settings": settings})
